doc/source/building_images/build_expandable_disk.rst

```diff
@@ -393,6 +393,15 @@ oemconfig.oem-unattended Element
   possible target devices are discovered the image is deployed to
   the first device. ``kiwi_oemunattended`` in the initrd
 
+oemconfig.oem-unattended-id Element
+  The target disk device for the installation is selected according
+  to the specified device ID. The device ID corresponds to the name of
+  the device as it exists for the configured `devicepersistency`. By
+  default this is the `by-uuid` device name. If no such representation
+  exist e.g for ramdisk devices, the unix device node can be used
+  to select. The given name must be present in the device list detected
+  by kiwi.
+
```

Comment:
```suggestion
.. warning::
    Setting this parameter will perform the installation without any confirmation whatsoever.

```

Reply:
Not sure if this is the the proper rst format tho (never used it but it seems like it supports it as per https://docutils.sourceforge.io/docs/ref/rst/directives.html#warning)